File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.java
class Solution {
    public int[] solution(String s) {
        int[] answer = new int[2];
        
        while (true) {
            int cnt = s.length();
            s = s.replace("0","");
            int c = s.length();
            
            answer[0]++;
            answer[1] += cnt - c;
            
            
            s = Integer.toString(c, 2);

            if (s.equals("1")) break;

        }
        
        
        return answer;
    }
}

File: 프로그래머스/2/70129. 이진 변환 반복하기/이진 변환 반복하기.py
def solution(s):
    answer = []
    
    a = 0
    b = 0
    
    while 1:
        if s == "1": break
        
        b += s.count("0")
        
        s = s.replace("0", "")
        
        s = bin(len(s))[2:]
        
        a += 1
    
    answer = [a,b]
    
    return answer